incorporating-accessibility-into-your-dev-process/demo/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Incorporating Accessibility Into Your Development Process</title>
+    <style>
+        html {
+            font-family: system-ui, sans-serif;
+        }
+
+        code {
+            font-family: ui-monospace, monospace;
+        }
+    </style>
+</head>
+
+<body>
+    <noscript>
+        You need to enable JavaScript to run this app.
+    </noscript>
+    <main></main>
```

Comment:
wont this main cause problems if you use a `main` inside the react?

Reply:
like the one in Forms

Reply:
This causes an error that accessibility tools pick up, so I want it like this for now.